components/ws-manager-mk2/controllers/subscriber_controller.go

```diff
@@ -80,23 +83,47 @@ func (r *SubscriberReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Ma
 		}
 	}()
 
+	filterByManagedBy := func(ws *workspacev1.Workspace) bool {
+		mgr, ok := ws.Labels[k8s.WorkspaceManagedByLabel]
```

Comment:
What happens when this gets deployed to an existing cell, how do we ensure active workspaces that don't yet have the managed-by label still get reconciled by mk2?

Reply:
We could check if it does not have a label. In that case case mk2 will take care of the workspace.

Reply:
Sounds good